examples/sops/local-configs/example-use.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
```

Comment:
Separate out this multi-object file into multiple files for clarity:

function.yaml
encrypted.yaml